public/main.js

```diff
@@ -52,32 +53,67 @@ function reloadTodoList() {
     getTodoList(function(todos) {
         todoListPlaceholder.style.display = "none";
         todos.forEach(function(todo) {
-            var listItem = document.createElement("li");
-            listItem.textContent = todo.title;
-            listItem.setAttribute("id", "item");
 
+            var row = document.createElement("li");
+            row.setAttribute("class", "list-group-item");
+
+            var listItem = document.createElement("div");
+            
+            // item text
+            var item = document.createElement("div");
+            item.textContent = todo.title;
+            item.setAttribute("id", "item");
+            if (todo.isComplete==="true") {
+                item.setAttribute("style", "display: inline-block;  text-decoration: line-through; font-style: italic; padding-left: 10px");
+            } else {
+                item.setAttribute("style", "display: inline-block;  text-decoration: none; font-style: normal; padding-left: 10px");
+            }
+
+            // delete button
             var deleteButton = document.createElement("button");
             deleteButton.setAttribute("type", "button");
             deleteButton.setAttribute("class", "btn btn-default");
             deleteButton.setAttribute("id", "deleteButton");
-
+            deleteButton.setAttribute("style", "right: 1%; position: absolute");
             var del = document.createElement("span");
             del.setAttribute("class", "glyphicon glyphicon-remove");
             del.setAttribute("aria-hidden", "true");
             deleteButton.appendChild(del);
 
 
+            // update button
             var updateButton = document.createElement("button");
             updateButton.setAttribute("type", "button");
             updateButton.setAttribute("class", "btn btn-default");
             updateButton.setAttribute("id", "updateButton");
-
+            updateButton.setAttribute("style", "right: 5%; position: absolute");
```

Comment:
The setup of the deleteButton and the updateButton are quite similar. Refactoring them to a separate function is a good idea.